ydb/library/yql/providers/generic/provider/yql_generic_load_meta.cpp

```
@@ -117,16 +117,19 @@ namespace NYql {
             YQL_ENSURE(State_->Configuration->ClusterNamesToClusterConfigs.cend() != it,
                        "cluster not found: " << tableAddress.ClusterName);
 
+            // preserve data source instance for the further usage
+            auto emplaceIt =
+                TableDescriptions_.emplace(tableAddress, std::make_shared<TTableDescription>());
+
+            auto desc = emplaceIt.first->second;
+
             NConnector::NApi::TDescribeTableRequest request;
-            FillDescribeTableRequest(request, it->second, tableAddress.TableName);
+            auto issues = FillDescribeTableRequest(request, it->second, tableAddress.TableName);
+            desc->Issues.AddIssues(issues);
```

У меня такое ощущение, что не совсем правильно прикапывать эту ошибку и позволять функции дальше выполняться, если она случилась. Ведь дальше пойдут уже сетевые вызовы, которые, получается, будут использовать невалидный запрос.

Может быть лучше будет fail fast подход, когда `LoadTableMetadataFromConnector` вернёт ошибку наружу, а `DoTransform` проверит её и вернёт наружу `TStatus::Error`?